recipes_source/recipes/module_load_state_dict_tips.py

```diff
@@ -0,0 +1,107 @@
+"""
+
+Tips for Loading an `nn.Module` from a Checkpoint
+=================================================
+
+In this tutorial, we will share some tips for loading a model from a checkpoint.
+In particular, we will discuss
+
+
+1.  The `mmap` keyword argument on `torch.load`
+2.  The `torch.device()` context manager
+3.  The `assign` keyword argument on `nn.Module.load_state_dict()`
+
+The following snippet of code illustrates the use of the above three utilities.
```

Comment:
I like the idea of having a "tl;dr:" at the top of the tutorial, we can make it more explicit: tl;dr: if you're loading a checkpoint and want to reduce compute and memory as much as posisble, do the following: